challenges/objects-arrays.js

```diff
@@ -59,11 +106,25 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+
+graduates.forEach((graduates) => {
+  let firstName = graduates.first_name;
+  let email = graduates.email;
+  return contactInfo.push(`${firstName} ${email}`);
+})
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+
+graduates.filter((graduates) => {
```

Comment:
Your current answer is an array of 10 elements which all say `false`.  The reason is that `===` is a strict equality operator and it returns a boolean.  All the elements are not 100% like "Uni" so it always returns false.  I see that you find the `toLowerCase()` method.  Try googling to see if there's another string method that can see if a string contains another part of a string etc.